core/shared/src/test/scala/com/dwolla/consul/ConsulServiceDiscoveryAlgSpec.scala

```diff
@@ -1,7 +1,7 @@
-package com.dwolla.http4s.consul
+package com.dwolla.consul
 
 import cats.syntax.all._
-import com.dwolla.http4s.consul.arbitraries._
+import com.dwolla.consul.arbitraries._
```

Comment:
This test is failing locally with the following error:
```
X com.dwolla.consul.ConsulServiceDiscoveryAlgSpec.Consul service lookup URI construction 2.78s java.time.zone.ZoneRulesException: Unknown time-zone ID: America/Chicago
    at java.time.zone.ZoneRulesProvider$.getProvider(/Users/dphumbhra/dev/http4s-consul-middleware/core/js/target/scala-2.13/http4s-consul-middleware-test-fastopt/main.js:13319)
    at java.time.zone.ZoneRulesProvider$.getRules(/Users/dphumbhra/dev/http4s-consul-middleware/core/js/target/scala-2.13/http4s-consul-middleware-test-fastopt/main.js:13357)
```
Not entirely sure where the timezone stuff is coming from because the test is just building up a uri 🤔